sw/device/tests/BUILD

```diff
@@ -272,6 +272,11 @@ opentitan_functest(
 opentitan_functest(
     name = "aon_timer_irq_test",
     srcs = ["aon_timer_irq_test.c"],
+    targets = [
```

Comment:
The "cw310_rom" test fails with the changes in `rom_start.S`. @alphan Do you think this deserves an issue and a TODO comment?